### ISSUE TITLE
Activity page - General Stationary Combustion (other than non-compression and non-processing)

### ISSUE THREAD

**Issue by dleard** [bcgov/cas-reporting#302]
#### Description:

[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7837-20066&t=WU1VPKoUDpbR5KCv-4): 

Following #253 we have a full stack example of how to generate an activity emissions form using the API form builder service.The implementation here will be really similar to GSC - without line tracing. Most of the needed database records will already exist.

Documentation on the full stack steps necessary to create an activity page are in our [docs](https://github.com/bcgov/cas-registration/tree/develop/docs/frontend)

Use the excel file (BCIERS Reporting Fields V2 in the SME channel files  - Reporting App directory) provided by IRC to determine what the necessary activity, source type, gas type, methodology, reporting fields & related configurations that need to be added are.

#### Acceptance Criteria:

Given I am on the GSC other than non-compression activity form page
When I first land on the page
Then the activity schema is fetched & displayed

Given the schema has rendered correctly
When I make selections that have dependencies (gas type, methodology)
Then the correct options and fields as defined from the configuration are shown

#### Development Checklist:

- [x] Any activities, source types, gas types, methodologies or reporting fields needed for the configuration that are not already in the database are added
- [x] The configuration relationships have correctly been added to the database
- [x] The many to many reporting field data (if needed) has correctly been added to the database
- [x] The base schema for the activity & source types have been added to the database
- [x] A routing directory/page has been added to both the industry_user & industry_user_admin flows for the activity
- [x] A parent component (named for the activity) has been added to the components in the activity directory
- [x] The page fetches the correct initial activity schema from the API service on initial load
- [x] A uiSchema exists in the components/activities/uiSchema directory for the activity & source types (will move to the database eventually, but can live here for now)
- [x] Conditional fields render dynamically as data is entered
- [x] On selecting a source type (or if there is only one, on loading the page) The first set of data fields is displayed (defaultEmptySourceTypeState)
- [ ] Matches the wireframe

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by ayeshmcg** [comment]
@suhafa  

Url for testing :
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/reporting/reports/5/facilities/1/activities/gsc_other_than_non_compression

Adding the fuel dropdowns was waiting on a discussion about the emission categories, fuels & how to apply that relationship. As a result, these dropdowns will not be visible on the activity page for now.

**Comment by suhafa** [comment]
Thanks @ayeshmcg for your work on this! Design review time:
1. This has to do more with the UI, but noticing that emissions field area is offsetting the solid fill towards the right side and jutting out like so: 

![Image](https://github.com/user-attachments/assets/0953c86d-8886-48e8-b272-ee4a075fab32)
We can remove the solid fill from the Emissions element to not have this issue? Not a priority

2. In the task list, the 'Mobile Combustion' is highlighted instead of 'General stationary combustion (other than non-compression and non-processing). Could just be a routing thing?


![Image](https://github.com/user-attachments/assets/580f81ef-e2c2-46d7-b000-c62dbf72a6ad)

3. For the bread crumbs, the activity shows up with '_' as a placeholder for space. Can we have it instead say 'GSC: other than non-compression and non-processing combustion'

![Image](https://github.com/user-attachments/assets/f12555c9-822a-4329-ab6a-4d82b1cab776)

**Comment by ayeshmcg** [comment]
Thanks, @suhafa
1. I can implement a quick fix for the first issue.
2. The sync between the task list and the multi-step form needs to be addressed separately. Currently, this issue affects all the activity pages.
3. Regarding the breadcrumbs, they currently reflect the URL path. We can modify the component to display custom text as needed.

**Comment by ayeshmcg** [comment]
@suhafa  
I have added the fix for spacing in the activity page.

**Comment by patriciarussellCAS** [comment]
Thanks @ayeshmcg - it looks like Dylan may have also made a ticket for this; #319 
Can you confirm that we can close this? Or should we keep 319 open in case the problem occurs again?

**Comment by ayeshmcg** [comment]
@patriciarussellCAS  I think we can close this one, as we already have https://github.com/bcgov/cas-reporting/issues/319.
